salt/metalk8s/deployed.sls

```diff
@@ -4,10 +4,10 @@ include:
   - metalk8s.kubernetes.coredns.deployed
   - metalk8s.repo.deployed
   - metalk8s.salt.master.deployed
-  - metalk8s.addons.ui.deployed
```

Comment:
For me you should not remove this from here just put it in the right order here and if needed add a logic in the `prometheus` state to check the status of the ConfigMap before finishing the state.

Maybe just add a state in `prometheus.deployed` that retrieve the configmap from k8s api with a retry just after the configmap deployement

Reply:
@jbertran ok you just have to add endpoint like `salt-master` and `repositories` in `pillar_data` in `salt/metalk8s/orchestrate/bootstrap/init.sls`